config/src/config/mempool_config.rs

```diff
@@ -103,19 +103,25 @@ impl ConfigOptimizer for MempoolConfig {
 
         // Change the default configs for VFNs
         let mut modified_config = false;
+        if node_type.is_validator() {
+            // Set the max_broadcasts_per_peer to 2 (default is 20)
+            if local_mempool_config_yaml["max_broadcasts_per_peer"].is_null() {
```

Comment:
This isn't used anymore because there is no mempool broadcast between validators with Quorum Store. For the time being, I want to preserve behavior as much as possible just in case we rollback Quorum Store.

Reply:
Can you add a comment to clarify this?